src/services/targets.service.ts

```diff
@@ -83,6 +83,25 @@ export class TargetsService {
     }
   }
 
+  async updateAwaitingCron( ): Promise<UpdateResult> {
+    try {
+      const updateResult = await this.targetRepository.updateAwaitingCron();
+      return updateResult;
+    } catch ( error ) {
+      throw new DatabaseError(`${error}`);
```

Comment:
by doing `${error}` you are parsing the error to string. What if error is a complex object?